src/bundle/Resources/public/js/scripts/admin.table.js

```diff
@@ -1,5 +1,5 @@
 (function(global, doc, $, eZ) {
-    const tables = doc.querySelectorAll('.ibexa-table');
+    const tablesWithAutomaticCheckbox = doc.querySelectorAll('.ibexa-table:not(.ibexa-table--no-automatic-main-checkbox)');
```

Comment:
Also instead of `.ibexa-table--no-bulk-checkbox` use `.ibexa-table--has-bulk-checkbox` to align with cell--has-checkbox (also discuessed in private)